room/runtime/src/main/java/androidx/room/RoomDatabase.java

```diff
@@ -1119,4 +1218,19 @@ public void onOpen(@NonNull SupportSQLiteDatabase db) {
         public void onDestructiveMigration(@NonNull SupportSQLiteDatabase db){
         }
     }
+
+    /**
+     * Callback for {@link Builder#createFromAsset(String)}, {@link Builder#createFromFile(File)}
+     * and {@link Builder#createFromInputStream(Callable)}
```

Comment:
We should add a bit more docs here: "This callback will be invoked after the pre-package DB is copied but before Room had a chance to open it and therefore before the RoomDatabase.Callback methods are invoked. This callback can be useful for updating the pre-package DB schema to satisfy Room's schema validation."